nvidia-gpu/vars.yaml

```diff
@@ -8,10 +8,10 @@ CONTAINER_TOOLKIT_REF: d9de4a09b8fd51a46207398199ecfeb3998ad49d
 LIBNVIDIA_CONTAINER_VERSION: v1.11.0
 LIBNVIDIA_CONTAINER_REF: c8f267be0bac1c654d59ad4ea5df907141149977
 # renovate: datasource=github-tags extractVersion=^go(?<version>.*)$ versioning=loose depName=golang/go
-GOLANG_VERSION: 1.17.13
```

Comment:
only @frezbo knows whether it's safe to update :see_no_evil:

Reply:
yeh, we need to check if there's a new libnvidia-container update, then cross check with their makefiles. Also if the code compiles i think it's safe to update once we complete the nvidiai e2e tests.